### ISSUE TITLE
Resources should be able to grow/spread without a Blossom Tree

### ISSUE THREAD

**Issue by Dutchygamer** [OpenRA/OpenRA#9524]
In TS, even loose Tiberium patches were able to grow/increase in density and spread to nearby cells over time. Currently in OpenRA, this only works when a Blossom Tree is present.

**Comment by airetaM** [comment]
Yes in Ra was too i often putted some infantry into ore and had some growing spots.
in tiberium infantry dies so its okay i think

**Comment by hypercube33** [comment]
Also, tiberium that grew into trees would convert them to blossom trees (using the above method of planting units, including infantry, tanks, etc) in tiberium and letting it spread.)

**Comment by StexoO** [comment]
I used to "manage" loose patches to get blossom trees but I've always wondered if it was intended or a bug from Westwood, like they forgot to write the code checking if patches are connected to a blossom tree.

**Comment by hypercube33** [comment]
> I used to "manage" loose patches to get blossom trees but I've always wondered if it was intended or a bug from Westwood, like they forgot to write the code checking if patches are "connected" to a blossom tree.

It 100% was intended. I'm not sure if you never played CNC Tiberium Dawn, or followed the storyline of the game, but it was a part of the history of the game, and part of the game's dynamics.

Tiberium spreads. Its a cancer. A Virus. It'll grow and convert living things over to its side. People. Trees. Your harvester blows up in your base, now its a minefield.

Also a big thing to note, is that free-standing units didnt die in Tiberium in Dawn like they do now. This I'm not sure if I'd consider a bug that this is changed in OpenRA, but without the spreading built into the game, I am not sure 

Here is a cutscene in TD showing a crystal spreading into 3 pods of crystals:
https://www.youtube.com/watch?v=bImKihXUU6A&feature=youtu.be&t=289

Here is another cutscene showing tiberium taking trees over:
https://youtu.be/bImKihXUU6A?t=799

**Comment by StexoO** [comment]
My bad, I completely forgot that.

**Comment by Mistranger** [comment]
Done: https://github.com/OpenRA/OpenRA/pull/13428

**Comment by Mailaender** [comment]
and https://github.com/OpenRA/OpenRA/pull/13969 for the Tiberium tree transformation.
